### PR TITLE
another updated PR-3

### PR DESCRIPTION
**Pull request by tr0mbl3y**

### CODE REVIEW

**Comment by johnjacobkenny** [comment]
@tr0mbl3y there is a conflict, could you please resolve and make the commit 🙂

**Comment by tr0mbl3y** [comment]
done!

**Comment by proful** [comment]
2 approval?

**Comment by johnjacobkenny** [comment]
Conflict again, please get two reviews too

File: docs/TEAM.md [hunk 1]
```diff
@@ -11,10 +11,27 @@
 |[Yaksh Bariya](https://www.github.com/thunder-coding) from India|A 14 year old technology lover. I create amazing things using the knowledge I have gained so far. Currently, I want to explore more of JavaScript/TypeScript.|
 |[Nishank Navelkar](https://www.github.com/nishanknavelkar)|Im from India, Basic knoweledge of Python, C, C++, HTML, CSS,  Electrical engineer, Here to learn and Contribute.|
 
+
+
 ---
 
 <br/>
 
+- [Pranav Goel](https://github.com/pranavgoel29), I'm from India, Have basic knowledge of HTML, CSS, C++ and C, I'm currently learning Javascript.
```
Comment (pranavgoel29) [inline docs/TEAM.md:20]: You have re-entered some name here, remove them first and also follow the table format.

File: docs/TEAM.md [hunk 2]
```diff
@@ -11,10 +11,27 @@
...
+- Usha Kiran ([K!r@nD0n](https://twitter.com/ushakiran_m)), I am a student from India. I just started my OpenSource journey with TWINDLE. I am currently learning #JavaScript.
+
+- Eyram from Ghana, I have no experience,I started learning web development some months ago. I am participating to learn and also make twindle a success.
+
+- [Michael Osajeh](https://github.com/michaelcosj),
+  From Lagos, Nigeria.
+  Second year Computer Science Undergraduate,
+  Started learning web and mobile development this year.
+  
+- [tr0mbley](https://github.com/tr0mbl3y) :learning how things works! nothing to share much
```
Comment (pranavgoel29) [inline docs/TEAM.md:31]: you should write it as
| [tr0mbley](https://github.com/tr0mbl3y) | learning how things works! nothing to share much |

**Comment by pranavgoel29** [comment]
Please first sync your forked repository with the main repository too.

**Review by UnevenCoder — APPROVED** [review]
Looks fine & ready to merge

**Review by michaelcosj — COMMENTED** [review]
I think you should check lines 14, 18 and 21

**Comment by tusharkandpal** [comment]
Yes, please recheck it :)

**Review by thunder-coding — CHANGES_REQUESTED** [review]
Table is not looking proper

**Comment by johnjacobkenny** [comment]
Could someone review once more

**Review by ad3rinto — APPROVED** [review]
Looks okay now,

**Comment by johnjacobkenny** [comment]
I see lots of removed lines! 🙄

**Comment by tr0mbl3y** [comment]
oh no what is happening

**Comment by pranavgoel29** [comment]
> Could someone review once more

I think removed two people by mistake I'm helping him to get that solved.

**Review by sarathkumar6 — CHANGES_REQUESTED** [review]
Needs change. Please fix the lines removed.

**Comment by sarathkumar6** [comment]
@pranavgoel29  Thanks for assisting here.

**Review by pranavgoel29 — APPROVED** [review]
Good to go.

**Comment by Rakesh-4** [comment]
@tr0mbl3y can you please update sunny's small mistake here before this thing gets merged. There's a line gap between `India` and `and`. Please move that line to the previous line to retain the format.

Sunny was trying to rectify it but unfortunately, he couldn't. Thanks
Screen Shot 2020-10-17 at 10 32 00 pm

**Comment by pranavgoel29** [comment]
> @tr0mbl3y can you please update sunny's small mistake here before this thing gets merged. There's a line gap between `India` and `and`. Please move that line to the previous line to retain the format.
> 
> Sunny was trying to rectify it but unfortunately, he couldn't. Thanks
> Screen Shot 2020-10-17 at 10 32 00 pm

Fixed that too
@johnjacobkenny can you merge it now.

**Comment by tr0mbl3y** [comment]
yes! done thanks  pranavgoel29

**Comment by Rakesh-4** [comment]
> yes! done thanks pranavgoel29

Thanks man

**Review by tusharkandpal — APPROVED** [review]
Afterall its ready... Great work 👍👍

**Review by EYRAM565 — APPROVED** [review]

**Comment by johnjacobkenny** [comment]
Merged! Thanks everyone for coordinating and completing 🥳

**Comment by tr0mbl3y** [comment]
finally! the **mystery of merge conflict** finished
thanks guys!

**Pull request merged**